docs/developer-docs/latest/development/backend-customization/middlewares.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Backend customization - Middlewares - Strapi Developer Documentation
+description : …
+---
+
+<!-- TODO: update SEO -->
+
+# Middlewares customization
+
+::: strapi Different types of middlewares
+
+In Strapi, 2 middleware concepts coexist:
+
+- **Strapi middlewares** are [configured and enabled](/developer-docs/latest/setup-deployment-guides/configurations/required/middlewares.md) for the entire Strapi server application. These middlewares can be applied at the application level or at the API level. <br/>The present documentation describes how to implement them.<br/>Plugins can also add Strapi middlewares (see [Server API documentation](/developer-docs/latest/developer-resources/plugin-api-reference/server.md#middlewares)).
```

Comment:
Do we want to call these Strapi Middlewares or just Global? Mainly because users could install additional Koa Middlewares and apply them (as in not designed just for Strapi).